doc/DEMO_TESTNET.md

```diff
@@ -0,0 +1,327 @@
+For this testnet setup, the security is not the primary goal, so the
+process of generating and sharing keys is left as an exercise to the readers.
+
+The required secrets are
+
+**if you are a stakeholder**:
+
+- one bip32 xpriv and its xpub.
+- one noise private key and its public key
+
+**if you are a manager**:
+
+- one bip32 xpriv and its xpub.
+- one noise private key and its public key
+
+**if you are a stakeholder and a manager**
+
+- two bip32 xpriv and their xpubs (one key pair called the stakeholder
+  keys and the other one the manager keys)
+- one noise private key and its public key
+- one cosigner bitcoin private key with its public key
+- one cosigner noise private key and its public key
+
+You can generate bip32 key pair on bip32.org
+For a noise key pair and a cosigner bitcoin key pair you can do:
+
+`python3 -c 'import os;print(os.urandom(32).hex())'`
+
+### Docker (kinda optional)
+
+We need docker to spin up the `coordinatord`'s Postgre database.
+You can avoid using docker though, and just spin up a Postgre database yourself.
+
+## Downloading the repositories
+
+Let's start by creating a dedicated folder - this way if you don't like Revault it will be easier to erase everything :D
+
+```
+mkdir revault_tutorial
+cd revault_tutorial
+```
+
+### If you are a stakeholder:
+
+Download all the needed repositories: `cosignerd`, `revaultd`, `revault-gui`:
+```
+~/revault_tutorial$
+git clone -b 0.2 https://github.com/revault/cosignerd
+git clone -b 0.2 https://github.com/revault/revaultd
+git clone -b 0.2 https://github.com/revault/revault-gui
+```
+
+Then in each of the repositories:
+
+```
+cargo install --path .
+```
+
+Binaries will be installed in `~/.cargo/bin`.
+
+### If you are a manager:
+
+Download all the needed repositories: `revaultd`, `revault-gui`:
+```
+~/revault_tutorial$
+git clone -b 0.2 https://github.com/revault/revaultd
+git clone -b 0.2 https://github.com/revault/revault-gui
+```
+
+Then in each of the repositories:
+
+```
+cargo build
+```
+
+## Spinning up Bitcoin Core
+
+Start a Bitcoin Core node using
+
+```
+bitcoind -testnet -daemon
+```
+
+## Starting the coordinator
+
+As we said, we need just one coordinator running, no matter how many stakeholders/managers there are.
+We'll properly update the coordinator configuration later. For now, let's just retrieve the coordinator's noise key.
+Cd into the coordinator:
+
+```
+cd coordinatord
+```
+
+Coordinatord needs a Postgre database running, we'll spin it up using docker:
+```
+docker run --rm -d -p 5432:5432 --name postgres-coordinatord -e POSTGRES_PASSWORD=revault -e POSTGRES_USER=revault -e POSTGRES_DB=coordinator_db postgres:alpine
+
+```
+
+Now duplicate the config provided:
+
+```
+cp contrib/config.toml coordinatord_config.toml
+```
+
+Make sure that the `postgres_uri` is `postgresql://revault:revault@localhost:5432/coordinator_db`.
+change the noise keys with the public noise keys of the participants.
```

Comment:
Well we still don't know the public noise keys of the participants... We should start the daemons before?

Reply:
Ah, but we can't start the daemon as it's not installed yet! Then we should leave here the dummy keys and properly update the conf afterwards

Reply:
User starts the installer which ask the private noise key, we expect that every secrets have been generated before starting the servers

Reply:
Yeah but you never explain how to get  the public noise key from the private...